src/Symfony/Component/Lock/Store/RedisStore.php

```diff
@@ -29,7 +29,7 @@ class RedisStore implements SharedLockStoreInterface
 {
     use ExpiringStoreTrait;
 
-    private const NO_SCRIPT_ERROR_MESSAGE = 'NOSCRIPT No matching script. Please use EVAL.';
+    private const NO_SCRIPT_ERROR_MESSAGE = 'NOSCRIPT No matching script.';
```

Comment:
Maybe rename the constant to `NO_SCRIPT_ERROR_MESSAGE_PREFIX` now?

Reply:
Perfect suggestion :+1: